### PR TITLE
Make Camera.setView options consistent with Camera.flyTo

### PR DESCRIPTION
**Pull request by hpinkos**
For #3088

* Changed `Camera.setView` to take the same arguments as `Camera.flyTo`
* Default heading/pitch/roll for `Camera.setView` was changed to be looking straight down.
* When `options.duration <= 0` in `Camera.flyTo`, camera calls `Camera.setView`
* General code cleanup for functions related to `setView`, `viewRectangle` and `flyTo`
* Deprecated `Camera.viewRectangle`

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -15,11 +15,17 @@ Change Log
   * Deprecated `PerspectiveFrustum.getPixelSize`, use `PerspectiveFrustum.getPixelDimensions` instead. It will be removed in 1.17.
   * Deprecated `PerspectiveOffCenterFrustum.getPixelSize`, use `PerspectiveOffCenterFrustum.getPixelDimensions` instead. It will be removed in 1.17.
   * Deprecated `Scene\HeadingPitchRange`, use `Core\HeadingPitchRange` instead. It will be removed in 1.17.
+  * Deprecated `Camera.setView`, use `EllipsoidGeometry` or `Entity.ellipsoid` instead. It will be removed in 1.17.
```
Comment (pjcozzi) [inline CHANGES.md]: Submit an issue for the deprecated calls.

**Comment by pjcozzi** [comment]
@bagnell can you review this?

File: Source/Scene/Camera.js [hunk 2]
```diff
@@ -210,7 +212,7 @@ define([
         this._max2Dfrustum = undefined;
 
         // set default view
-        this.viewRectangle(Camera.DEFAULT_VIEW_RECTANGLE, scene.mapProjection.ellipsoid);
+        rectangleCameraPosition3D(this, Camera.DEFAULT_VIEW_RECTANGLE, this.position, true);
```
Comment (hpinkos) [inline Source/Scene/Camera.js]: @bagnell using `this.setView` here breaks a whole bunch of the tests and I couldn't figure out why.

**Comment by hpinkos** [comment]
@bagnell this is ready for review now.  i finished making the changes i wanted to.

**Pull request merged**